fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/AvatarCarousel.kt

```diff
@@ -55,6 +55,22 @@ private fun Modifier.clickAndSemanticsModifier(
             role = Role.Button
         )
 }
+private fun updateSemantics(item: AvatarCarouselItem, enablePresence: Boolean): Modifier {
+    return if (item.contentDescription != null) {
+        Modifier.clearAndSetSemantics{
+            this.contentDescription = item.contentDescription
+        }
+    } else {
+        Modifier.clearAndSetSemantics{
+            this.contentDescription = "${item.person.getName()}. " +
+                    "${if (enablePresence) "Status, ${item.person.status}," else ""} " +
+                    "${if (enablePresence && item.person.isOOO) "Out Of Office," else ""} " +
+                    if (item.enableActivityRing) {
```

Comment:
nit: this need not be a if else check - it can be a simple check for `if (item.enableActivityRing)` as the else condition is no-op. Similar for the checks above on `enablePresence`